images/marketplace-operator.yml

```diff
@@ -8,7 +8,7 @@ content:
     git:
       branch:
         target: release-{MAJOR}.{MINOR}
-      url: git@github.com:operator-framework/operator-marketplace.git
+      url: git@github.com:openshift-priv/operator-marketplace.git
```

Comment:
this ought to be correct, but for it to work, we need to add an upstream_source specifically for this, and also the fast-forwarding needs to be fixed so the right branches are there. so for now, omit this.